packages/eds-core-react/src/components/Typography/Typography.docs.mdx

```diff
@@ -0,0 +1,75 @@
+import { Links, PropsTable, Story, Stack, TypographyGroups } from './../../../.storybook/components'
+import { Typography } from '.'
+
+# Typography
+
+Presents hierarchy and organises information as clearly and efficiently as possible, <br />
+therefore text sizes and styles were developed to balance content density and reading comfort.
+
+<Links 
+    figmaUrl="https://www.figma.com/file/0TbIXrrObWj80Cf7KucKYFL0/Design-Tokens?node-id=2%3A4"
+    documentationUrl="https://eds.equinor.com/0b0c666ab/p/721544-typography/b/32c137"
+    sourceUrl="https://github.com/equinor/design-system/blob/develop/packages/eds-core-react/src/components/Typography/Typography.tsx"
+    npmUrl="https://www.npmjs.com/package/@equinor/eds-core-react"
+></Links>
+
+<Story id="typography-typography--introduction" />
+<PropsTable />
+
+## Usage
+
+**Typeface**
+
+The `Equinor` typeface is the primary typeface and is available in four weights:
+
+<ul>
+  <li><code>light</code>,</li>
+  <li><code>regular</code>,</li>
+  <li><code>medium</code>,</li>
+  <li><code>bold</code>,</li>
+  <li>with accompanying <code>italics</code>.</li>
+</ul>
+
+Please do not use <code>light</code> in digital interfaces except in special cases where the font size is over <code>48px</code>.
+
```

Comment:
Perhaps a note about how the equinor font is not included in eds by default and has to be added (and how)

Reply:
<div style="padding-top:75.362%;position:relative;"><iframe src="https://gifer.com/embed/92Ul" width="100%" height="100%" style='position:absolute;top:0;left:0;' frameBorder="0" allowFullScreen></iframe></div><p><a href="https://gifer.com">via GIFER</a></p>

Reply:
![Alt Text](https://i.gifer.com/92Ul.gif)

Reply:
done